azext_edge/tests/edge/checks/int/test_akri_int.py

```diff
@@ -0,0 +1,108 @@
+# coding=utf-8
+# ----------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License file in the project root for license information.
+# ----------------------------------------------------------------------------------------------
+
+from typing import Any, Dict
+import pytest
+from knack.log import get_logger
+from azext_edge.edge.providers.check.common import ResourceOutputDetailLevel
+from azext_edge.edge.providers.edge_api import (
+    AkriResourceKinds, AKRI_API_V0
+)
+from .helpers import (
+    assert_enumerate_resources,
+    assert_eval_core_service_runtime,
+    assert_general_eval_custom_resources,
+    run_check_command
+)
+from ....helpers import get_kubectl_custom_items
+from ....generators import generate_names
+
+logger = get_logger(__name__)
+
+
+@pytest.mark.parametrize("detail_level", ResourceOutputDetailLevel.list())
+@pytest.mark.parametrize("resource_kind", AkriResourceKinds.list() + [None])
+# TODO: figure out if name match should be a general test vs each service (minimize test runs)
+@pytest.mark.parametrize("resource_match", [None, "*otel*", "akri-opcua*", generate_names()])
```

Comment:
leaning towards removing resource match testing here (and in opcua) and adding a todo for general resource match testing.